src/plugins/controls/common/control_group/types.ts

```diff
@@ -6,83 +6,66 @@
  * Side Public License, v 1.
  */
 
-import { EmbeddableInput, PanelState } from '@kbn/embeddable-plugin/common/types';
-import { SerializableRecord } from '@kbn/utility-types';
-import { ControlInput, ControlStyle, ControlWidth } from '../types';
+import { DataViewField } from '@kbn/data-views-plugin/common';
+import { ControlStyle, DefaultControlState, ParentIgnoreSettings } from '../types';
 
 export const CONTROL_GROUP_TYPE = 'control_group';
 
-export interface ControlPanelState<TEmbeddableInput extends ControlInput = ControlInput>
-  extends PanelState<TEmbeddableInput> {
-  order: number;
-  width: ControlWidth;
-  grow: boolean;
-}
-
 export type ControlGroupChainingSystem = 'HIERARCHICAL' | 'NONE';
 
-export interface ControlsPanels {
-  [panelId: string]: ControlPanelState;
+export type FieldFilterPredicate = (f: DataViewField) => boolean;
```

Comment:
These types were simply moved from `public` to `common` so that they can be used in server-side code where necessary. They are **not** new types.